src/modules/tests/test_data/test-data.reducer.ts

```diff
@@ -33,6 +33,22 @@ export function testDataReducer(
           [action.payload]: 'DF',
         },
       };
+    case testDataActions.ADD_MANOEUVRE_SERIOUS_FAULT:
+      return {
+        ...state,
+        manoeuvres: {
+          ...state.manoeuvres,
+          [action.payload]: 'S',
+        },
+      };
+    case testDataActions.ADD_MANOEUVRE_DANGEROUS_FAULT:
+      return {
+        ...state,
+        manoeuvres: {
+          ...state.manoeuvres,
+          [action.payload]: 'D',
```

Comment:
Would be nice to have a more descriptive name for the parameter on these actions.

Reply:
True, but the type of the `payload` suggests what is it going to be. Also, all `test-data.actions` have `payload` as payload name